assets/scripts/utils/maths.js

```diff
@@ -31,9 +31,22 @@ const lerp = (x, y, a) => x * (1 - a) + y * a
 const invlerp = (x, y, a) => clamp((v - x)/(a - x))
 
 
-// Export
+/**
+ * Round number to n decimal
+ * @param {number}  x   - float number to round
+ * @param {number}  n   - integer number of decimals
+ * @return {number} rounded number
+ */
+
+const roundNumber = (x, n = 2) => {
+    const decimals = 10 ** n
+    return Math.round(x * decimals)/decimals
+}
```

Comment:
I suggest using [`Number.prototype.toPrecision()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision) instead.

```suggestion
```

Reply:
@lucasvallenet 👋

Reply:
I suggest keeping this function because it returns a number, and [Number.prototype.toPrecision()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision) returns a string 🤷‍♂️

Reply:
Oh! I did not know that.

In that case, what about this:

```suggestion
/**
 * Round number to the specified precision.
 *
 * This function is necessary because `Number.prototype.toPrecision()`
 * and `Number.prototype.toFixed()`
 *
 * @param  {number}  number      - The floating point number to round.
 * @param  {number}  [precision] - The number of digits to appear after the decimal point.
 * @return {number} The rounded number.
 */
const roundNumber = (number, precision = 2) => {
    return Number.parseFloat(number.toPrecision(precision));
}
```